### PR TITLE
Merge vagrant changes into master

### PR DESCRIPTION
**Pull request by damonp**

### CODE REVIEW

File: Berksfile.lock [hunk 1]
```diff
@@ -123,7 +142,7 @@ GRAPH
   python (1.4.6)
     build-essential (>= 0.0.0)
     yum-epel (>= 0.0.0)
-  rabbitmq (3.3.0)
+  rabbitmq (3.9.0)
```
Comment (jmetzmeier) [inline Berksfile.lock]: This version of the rabbitmq did not work on the version of chef opsworks runs when I tried it.

Comment (alexlovelltroy) [inline Berksfile.lock]: What was the error?

Comment (jmetzmeier) [inline Berksfile.lock]: It was using a function introduced in a newer version of Chef (or maybe it was a type attribute). Unfortunately, I don't recall what the function it was.

**Comment by alexlovelltroy** [comment]
I tried this again and got a similar error, but changing it to a `bash` resource didn't fix it and instead led to a similar error in the `bash` resource.  This should work and I don't know why it doesn't.  We need another chef-head to poke at it.

**Comment by jonathandietz** [comment]
This is a pretty big merge and it doesn't look like it'll merge cleanly with Master anymore. @damonp can you resync master with your branch here?

**Comment by jonathandietz** [comment]
I've made a new pull request (#41) to merge these vagrant changes into Master.

**Pull request merged**